skbeam/io/tests/test_fit2d_save.py

```diff
@@ -44,16 +44,18 @@
 import numpy as np
 from skbeam.io.fit2d import fit2d_save, read_fit2d_msk
 from numpy.testing import assert_array_equal
+from tempfile import tempdir
 
 
 def test_save_output_fit2d():
+    tdir = tempdir()
```

Comment:
I think you want `TemporaryDirectory`. `tempdir` is the cached result of the most recent call to `tempfile.gettempdir()`, so it is typically a value like `None` or `/tmp`, not a callable. Given that this is somehow passing Travis, I worry that the tests are still not being run. Please test the code locally with print statements to be sure it is actually executed.